src/Engines/AlgoliaEngine.php

```diff
@@ -40,7 +40,7 @@ public function __construct(Algolia $algolia, $softDelete = false)
     /**
      * Update the given model in the index.
      *
-     * @param  \Illuminate\Database\Eloquent\Collection  $models
+     * @param  \Illuminate\Database\Eloquent\Collection $models
```

Comment:
I guess this should be reverted too. The spaces come from the code style which is used for docblocks by laravel.